content/2017.03.building-projects-as-a-distributed-team.rst

```diff
@@ -0,0 +1,178 @@
+Building software as a distributed team
+#######################################
+
+:slug: building-software-as-a-distributed-team
+:date: 2017-03-15
+:lang: en
+:url: building-software-as-a-distributed-team
+:summary:
+   How do you build software together when you don't have overlapping
+   working hours?
+
+During the last few months we moved from a team mostly based in the
+Paris timezone to a team distributed across 8 timezones.
+
+- 4 people are in France and Spain (Niko, Mago, Mat and Natim)
+- 1 person is in New-York City (Ethan)
+- 1 person, our manager, is in Vancouver (Ben)
+
+
+Working across timezones
+========================
+
+We have only 2 time slots available to have a meeting alltogether:
+
++------------+----------+-------+
+| Vancouver  | New-York | Paris |
++------------+----------+-------+
+|   06:00    |  09:00   | 15:00 |
++------------+----------+-------+
+|   08:00    |  11:00   | 17:00 |
++------------+----------+-------+
+|   11:00    |  14:00   | 20:00 |
++------------+----------+-------+
+|   14:00    |  17:00   | 23:00 |
++------------+----------+-------+
+|   16:00    |  19:00   | 01:00 |
++------------+----------+-------+
+|   23:00    |  02:00   | 08:00 |
++------------+----------+-------+
+
+As you can see that's not a lot of overlap, which means we need to be
+able to collaborate asynchroneously.
+
+The process of building software in a remote team
+=================================================
+
+Working in a remote team is really similar to working with a
+community.
+
+You need to explain everything you do, you need to find consensus,
+you need to take feedback, you need to propose solutions and try
+things.
+
+As a team, as a community, you are stronger because instead of being
+alone you can rely on others to help you make the best choices and
+write the best software.
+
+
+The Prototype
+=============
+
+You've got an idea, let's make it a prototype.
```

Comment:
"Let's make it a prototype" makes it sound like you're saying something like "Suppose you have an idea, let's say it's [already] a prototype". Maybe something like `Once you have an idea, you turn it into a prototype.`